src/App.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+body {
+  font-family: gillsans, sans-serif;
+}
+
+.font-lato {
+  font-family: lato, sans-serif !important;
+}
+
+.text-custom1 {
+  color: #4369b2 !important;
+}
+
+.bg-custom1 {
+  background-color: #4369b2 !important;
+}
+
+.bg-custom2 {
+  background-color: #5176be !important;
+}
+
+.bg-custom3 {
+  background-color: #3c5e9f !important;
+}
+
+.cursor-pointer {
+  cursor: pointer !important;

```

Comment:
[Optional] This is not a CSS best practice, Kindly avoid excessive use of `!important`